### DISCUSSION TITLE
Trackers / block explorers for Local chains

### DISCUSSION THREAD

**Discussion by paulrouge** [icon-project/btp2#17] [General]
For me it will speed up my workflow to have trackers / block explorers in the browser.

I am now using https://tryethernal.com/ for the evm chain running in the docker container running via the e2e-demo. Seems to be working fine. Make sure though, that your browser isnt blocking pop ups and stuff for the site (in Brave: put the shields down by clicking the little lion).

I havent been able to have a tracker instance hooked to a local Icon chain, which would be quite helpful, any one having issues with this as well?

**Comment by paulrouge** [comment]
The above mentioned tryethernal tracker is not ideal. Been searching for other solutions, suggestions are very welcome.

**Comment by hemz10** [comment]
I agree, I too was looking for a block explorer that would work with local chain unfortunately was not able to find one. 
Having one will definitely speed up the process of testing in local chain and also will have access to additional information like all transaction or events that has happened in a contract.
